client/components/TestRenderer/index.jsx

```diff
@@ -245,14 +246,26 @@ const TestRenderer = ({
         await testRunIO.setInputsFromCollectedTestAsync(renderableContent);
         testRunIO.setConfigInputFromQueryParamsAndSupport(configQueryParams);
 
+        // https://github.com/w3c/aria-at/commit/9d73d6bb274b3fe75b9a8825e020c0546a33a162
+        // This is the date of the last commit before the build folder removal.
+        // Meant to support backward compatability until the existing tests can
+        // be updated to the current structure
+        const buildRemovalDate = new Date('2022-03-10 18:08:36.000000 +00:00');
+        const testUpdatedAtDate = new Date(testUpdatedAt);
+
+        const finalTestPageUrl =
```

Comment:
Based on discussion with @alflennik, we may also be able to completely remove the dependency on the implicit commit attached to this date and reference the file directly from the repo's root. ie. `<proxy_domain>/tests/<testplan>/path/file.html` instead of `<proxy_domain>/<commit>/tests/<testplan>/path/file.html` but may require further discussion.